scripts/environments/local.env.example

```diff
@@ -120,6 +120,7 @@ export AZURE_OPENAI_EMBEDDINGS_MODEL_VERSION=""
 
 # If you have limited capacity in your subscription, you can set the following to limit the deployment capacity.
 export AZURE_OPENAI_CHATGPT_MODEL_CAPACITY="240" # Required
+export AZURE_OPENAI_EMBEDDINGS_MODEL_CAPACITY="240" # Required
```

Comment:
Please add documentation in `docs/deployment/deployment.md` to reflect this new local.env parameter.

Reply:
updated